network-api/networkapi/wagtailpages/utils.py

```diff
@@ -206,7 +206,7 @@ def insert_panels_after(panels, after_label, additional_panels):
     )
 
     if position is not None:
-        cut = position + 2
+        cut = position + 1
```

Comment:
Wondering why it was `position + 2` before and `position + 1` now? What are the offsets for?

Reply:
it was an off-by-one bug. It's the position of where we need to insert a collection of panels inside the parents' list of panels. The idea is to insert after whatever label is indicated, so it needs to find that index, and then cut the panels list _after_ that index. For some reason, we were cutting one index too far.

Reply:
ah got it! Thanks for the explanation!